lib/kitchen/directions/bake_notes/bake_numbered_notes.rb

```diff
@@ -26,6 +26,17 @@ def self.bake_note(note:)
             </div>
           HTML
         )
+
+        return unless note['use-subtitle']
+
+        bake_subtitle(note: note, title: note.title&.cut)
+      end
+
+      def self.bake_subtitle(note:, title:)
+        title.name = 'h4'
+        title.add_class('os-subtitle')
+        title.wrap_children('span', class: 'os-subtitle-label')
+        note.first!('.os-note-body').prepend(child: title.to_s)
```

Comment:
@jpslav where would be a good place to move the bake_subtitle method so that it could be accessed by both `BakeNumberedNotes` and `BakeAutotitledNotes`?

Reply:
What about `BakeNoteSubtitle.v1(note:)` and move the `note.title&.cut` inside that direction?  Both `BakeNumberedNotes` and `BakeAutotitledNotes` seem to call that.